### PR TITLE
Added the station names and the lines connecting the names

### PR DESCRIPTION
**Pull request by ravernkoh**
Might want to check out Station.ts I added in a quick and dirty data structure. Might want to refactor; Integrate it into your Station objects.

### CODE REVIEW

File: public/src/js/Index.ts [hunk 1]
```diff
@@ -56,6 +57,69 @@ function createStation(station: Station, color: string) {
...
+        fontSize: 18,
+        fontFamily: 'sans-serif',
+        fontStyle: 'bold',
+        width: 110,
+        padding: 0,
+        align: 'center',
+    });
+    text.setOffset({
+        x: text.getWidth() / 2,
+    });
```
Comment (omfgshaz) [inline public/src/js/Index.ts]: `setOffset()` does not exist on `text`. Use `text.offsetX()` instead.

**Comment by omfgshaz** [comment]
Stations has a dirty structure from the start so don't worry about it. I will clean up after Open House.

**Comment by ravernkoh** [comment]
Strange. The call to setOffset works but it says it doesn’t exist.

**Comment by ravernkoh** [comment]
Will change it later

**Pull request closed without merging**